pages/components/input.html

```diff
@@ -106,8 +106,60 @@ <h1 clss="page-title">input</h1>
 
 
   <main class="main-content">
-
-    <iframe src="/codeblock/x.txt" frameborder="0" class="codeblock"></iframe>
+    <form action="#" class="form">
+        <div class="form-item status-success-outline">
```

Comment:
I think "outline" is redundant in the class name for status if we don't have any other type of status (besides outline)

Reply:
My status classes are background and outline and I have reused these classes everywhere like alert, badge etc. so I need to differentiate them